tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -455,6 +458,14 @@ setup_selinux() {
 
 setup_systemd() {
 		local mount_dir="$1"
+		if [ "${DM_VERITY}" == "no" ]; then
+			for u in "${udev_systemd_units[@]}"; do
```

Comment:
Why don't you simply append `udev_systemd_units` to `systemd_units` and let the loop on line 471 to remove everything?

Reply:
When installing dependencies with dm-verity, we need to install `udev`, and `udev`  need to be set as a startup process, so the related files of `udev` need to be preserved.